pageserver/src/repository.rs

```diff
@@ -19,8 +19,6 @@ pub type BlockNumber = u32;
 /// A repository corresponds to one .zenith directory. One repository holds multiple
 /// timelines, forked off from the same initial call to 'initdb'.
 pub trait Repository: Send + Sync {
-    fn shutdown(&self) -> Result<()>;
```

Comment:
Do not get it who is responsible for performing checkpoint when pageserver is shutting down. I see that shutdown_timeline code migrated to put_timelines_into_tenant but I do not see where we trigger this code when pageserver is stopped

Reply:
`shutdown_all_tenants`

Reply:
Got it, thanks!